docs/hub/spaces-sdks-docker-argilla.md

```diff
@@ -30,40 +30,81 @@ If you want to customize the title, emojis, and colors of your space, go to "Fil
 
 Once you have created the space, you'll see the `Building` status and once it becomes `Running` your space is ready to go. If you don't see the Argilla login UI refresh the page.
 
-You'll see the login screen where you need to use either argilla or team with the default passwords (1234). Remember you can change the passwords using secret variables. If you get a 500 error when introducing the credentials, make sure you have correctly introduce the password.
+The Space is configured with **two users**: **argilla** and **team** with the same default password: **1234**. If you get a 500 error after login, make sure you have correctly introduce the user and password. To secure your Space, you can change the passwords and API keys using secret variables as explained in the next section. 
+
+## Set up passwords and API keys using secrets (optional)
 
 <Tip>
-For quick experimentation, you can jump directly into the next section. If you want to add access restrictions, go to the "Setting up secret environment variables" at the end of this document. Setting up secret variables is recommended for longer-term usage.
+For quick experimentation, you can jump directly into the next section. If you want to secure your space and for longer-term usage, setting up secret variables is recommended.
 </Tip>
 
-### Create your first dataset
+The Space template can be configured with **optional settings** to secure your Argilla Space.
+
+You can configure secret variables in the Settings tab of your Space. Make sure to save these values somewhere for later use.
+
+The template space has two users: **team** and **argilla**. The **username team is the root user**, who can upload datasets and access any workspace within your Argilla Space. The **username argilla is a standard user** with access to the `team` workspace and its own workspace called `argilla`. 
+
+Currently, **the user names can't be configured**. The **passwords and API keys** to upload, read, update, and delete datasets **can be configured** using the following secrets:
+
+- `TEAM_API_KEY`: This sets the API key for the root user: team. You can use this API key for uploading data if you want to prevent the user argilla from deleting datasets in the UI. The API key can be any string of your choice.
```

Comment:
```suggestion
- `TEAM_API_KEY`: This sets the API key for the root user: `team`. You can use this API key for uploading data if you want to prevent the user `argilla` from deleting datasets in the UI. The API key can be any string of your choice.
```